pylintrc

```diff
@@ -0,0 +1,381 @@
+# ***************************
```

Comment:
I don't think you'll need to add this whole file..  Would recommend trying pylint without adding a `pylintrc`;  Their [default settings](https://github.com/PyCQA/pylint/blob/master/pylintrc) are good.  You can just fix whatever pylint complains about, if anything. It's a tiny repo, so we're not too concerned about keeping letter-perfect with [edx-lint](http://github.com/edx/edx-lint).